frontend/src/routes/login.lazy.tsx

```diff
@@ -65,18 +66,94 @@ function Login() {
             Log in
           </h1>
           <div className="card flex flex-col gap-3">
-            {data?.errors?.root && (
-              <div style={{ color: "red" }}>{data.errors.root}</div>
-            )}
-            <div>
+            <Form.Root>
```

Comment:
```suggestion
            <Form.Root asChild>
               <form action={/*...*/}>
```

Form field should wrap the `form` not be inside.